### DISCUSSION TITLE
Olat Support

### DISCUSSION THREAD

**Discussion by IwisGiwis** [SafeExamBrowser/seb-server#56] [Q&A] [answered]
Dear SEB team,
is SEB-Server supporting Olat? In specific Course API and SEB restriction API. In the manual you only mention open edX and Moodle, but I found this article https://www.uzh.ch/blog/zi/2021/11/22/seb-server-olat-integration-im-eilverfahren/ where it is written that there is/you are working on an Olat support. 

Best regards
Stefan

**Comment by anhefti** [comment, marked as answer]
Dear Stefan,

Thank you for your interests. 
SEB Server has an unofficial Olat integration since Version 1.2. The SEB Server Olat integration is fully developed by the Olat dev team and I propose to ask your question directly to them: [OpenOLAT UZH](https://www.zi.uzh.ch/en/teaching-and-research/software-elearning/olat.html)

Best regards
Andreas

**Reply by IwisGiwis** [reply]
Dear Andreas,

Thanks for the fast reply.

Best regards
Stefan
